### PR TITLE
RSDK-6664 correct docs

### PR DESCRIPTION
**Pull request by maximpertsov**
follow-up to https://github.com/viamrobotics/viam-python-sdk/pull/545

### CODE REVIEW

**Comment by npentrel** [comment]
thank you :)

**Review by purplenicole730 — APPROVED** [review]
LGTM, but just wanted to make sure: `GetCloudMetadata` returns a list of `GetCloudMetadataResponse`? In `test_robot.py`, 
`GET_CLOUD_METADATA_RESPONSE` only returns one response.

File: src/viam/robot/client.py [hunk 1]
```diff
@@ -632,7 +632,10 @@ async def stop_all(self, extra: Dict[ResourceName, Dict[str, Any]] = {}):
 
     async def get_cloud_metadata(self) -> GetCloudMetadataResponse:
         """
-        Returns app-related information about the robot.
+        Get app-related information about the robot.
+
+        Returns:
+            List[viam.proto.robot.GetCloudMetadataResponse]: App-related metadata.
```
Comment (maximpertsov) [inline src/viam/robot/client.py:638]:
```suggestion
            viam.proto.robot.GetCloudMetadataResponse: App-related metadata.
```

**Comment by maximpertsov** [comment]
> LGTM, but just wanted to make sure: `GetCloudMetadata` returns a list of `GetCloudMetadataResponse`? In `test_robot.py`, `GET_CLOUD_METADATA_RESPONSE` only returns one response.

no you're right - I copy-pasted bad :(

**Pull request merged**